fgpyo/fasta/tests/test_reference_set_builder.py

```diff
@@ -0,0 +1,11 @@
+"""Basic tests for reference_set_builder"""
+
+from fgpyo.fasta.reference_set_builder import ReferenceSetBuilder
+
+
+def test_add_seq_to_record() -> None:
+    builder = ReferenceSetBuilder()
+    builder.add("chr1").add("AAAAAAAAAA", 10).add("NNNNNNNNNN", 10)
+    builder.add("chr10").add("GGGGGGGGGG", 10)
+    assert len(builder.REF_BUILDERS[0].bases) == 200
+    assert len(builder.REF_BUILDERS[1].bases) == 100
```

Comment:
Just thinking as I look at this - probably it would be better for `REF_BUILDERS` to be "private" and implement `def __getitem__(self, key: Union[int, str])` to look up items by either index or name.  That way how the data is held internally is hidden from users and can be changed if needed.

Reply:
The method in commit cb8ac832a05ba4a3ad304edb2de19c57d6f9b8f3 takes only a string (name) as I changed  'REF_BUILDERS' list to `__contig_builders` dict. Happy to change back to a list!